src/core/usecase/usecase.test.ts

```diff
@@ -165,6 +175,45 @@ describe('UseCase', () => {
       );
     });
 
+    it('passes map variant and map revision', async () => {
+      const {
+        usecase,
+        performSpy,
+        profileConfiguration,
+        providerConfiguration,
+      } = createUseCase();
+
+      const rebindSpy = jest.spyOn(usecase as any, 'rebind');
+      await expect(
+        usecase.perform(
+          { x: 7 },
+          {
+            security: [
+              {
+                id: 'test',
+                apikey: 'key',
+              },
+            ],
+            mapRevision: 'rev',
+            mapVariant: 'var',
+          }
+        )
+      ).resolves.toBeUndefined();
```

Comment:
Yeah, this is because of mocking BoundProfileProvider without actual implementation - we test if correct values are passed to perform not what perform returns. It looks odd but I believe it is ok.

Reply:
Thought it is just mocked, and should spend a bit more on checking myself. Thank you.